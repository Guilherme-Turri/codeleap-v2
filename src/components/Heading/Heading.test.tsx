import { componentRender } from '../../render/componentRender'
import { Heading } from '.'
import { screen } from '@testing-library/react'

describe('<Heading />', () => {
  it('should render on screen with h1 and uppercase true', () => {
    componentRender(<Heading big={true} as='h1' uppercase={true}>test</Heading>)
    expect(screen.getByText('test')).toHaveStyle('text-transform: uppercase');
  })
  it('should render on screen with h2 and uppercase false', () => {
    const { container } = componentRender(<Heading big={false} as='h2' uppercase={false}>test</Heading>)
    expect(screen.getByText('test')).toHaveStyle('text-transform: none');
    expect(container).toMatchSnapshot();
  })
})