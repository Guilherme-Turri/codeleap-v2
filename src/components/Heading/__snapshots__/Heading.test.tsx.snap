// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> should render on screen with h2 and uppercase false 1`] = `
.c0 {
  text-transform: none;
  color: #00CBA9;
  font-size: 3rem;
  font-family: Nanum Brush Script;
}

<div>
  <h2
    class="c0"
  >
    test
  </h2>
</div>
`;
